### PR TITLE
kde-apps/kde-l10n: Block <=kde-apps/kwave-0.9.2 for file collisions

### PR DESCRIPTION
**Pull request by a17r**
Package-Manager: portage-2.3.0

### CODE REVIEW

File: kde-apps/kde-l10n/kde-l10n-16.12.0.ebuild [hunk 1]
```diff
@@ -37,6 +37,7 @@ RDEPEND="
 	!kde-apps/ktp-l10n
 	!kde-apps/ktp-send-file:4
 	!kde-apps/ktp-text-ui:4
+	!<=kde-apps/kwave-0.9.2
```
Comment (johu) [inline kde-apps/kde-l10n/kde-l10n-16.12.0.ebuild]: How about !<kde-app/kwave-15.11.50?

Comment (a17r) [inline kde-apps/kde-l10n/kde-l10n-16.12.0.ebuild]: There wasn't any version like that, though?

Comment (johu) [inline kde-apps/kde-l10n/kde-l10n-16.12.0.ebuild]: Yes, but it indicates when the proj was added to kde apps and makes blocker cleanup easier.

**Comment by johu** [comment]
Thanks!

**Pull request closed without merging**